lib/Mojo/Date.pm

```diff
@@ -69,6 +71,16 @@ sub to_string {
     $MONTHS[$month], $year + 1900, $h, $m, $s;
 }
 
+# ($t, $f) = _modf($x)
```

Comment:
You have to be careful with POSIX because it is not dual life, so some functions may not be available in old perl versions. I have verified that floor() is available in 5.10.1. Just make sure to specify an import list for POSIX (even an empty one): https://metacpan.org/pod/Perl::Critic::Policy::Freenode::POSIXImports